forte/api/forte_python_module.cc

```diff
@@ -141,6 +143,7 @@ PYBIND11_MODULE(forte, m) {
     m.def("make_fragment_projector", &make_fragment_projector,
           "Make a fragment(embedding) projector");
     m.def("make_embedding", &make_embedding, "Apply fragment projector to embed");
+    m.def("build_aset2_fragment", &build_aset2_fragment, "Generate fragment (A) MOSpaceInfo object");
```

Comment:
I'm going to request this function be renamed, both Py-side and C-side to something that makes clear you are just generating the `MOSpaceInfo`, not the embedding Hamiltonian.

Reply:
I change it to build_aset2_mospaceinfo.